Kernel_23/doc/Kernel_23/CGAL/Aff_transformation_3.h

```diff
@@ -200,12 +200,12 @@ returns `true`, if the transformation is reflecting.
 bool is_odd() const;
 
 /*!
-returns `true`, if the transformation type is the specialized scaling.
+returns `true`, if the object was constructed using the tag `CGAL::Scaling`, or as a composition of such objects.
```

Comment:
```suggestion
returns `true`, if the object was constructed using the tag `CGAL::Scaling`, or is the result of the composition of such objects only.
```

Reply:
not sure about 'only' on the end I'd put:
>returns `true`, if the object was constructed using the tag `CGAL::Scaling`, or is the result of the composition of only scaling transformations.